src/__swaps__/utils/gasUtils.ts

```diff
@@ -14,24 +15,70 @@ import { ParsedAsset } from '@/__swaps__/types/assets';
 import { ChainId } from '@/chains/types';
 import { BlocksToConfirmation, GasFeeLegacyParams, GasFeeParam, GasFeeParams, GasSpeed } from '@/__swaps__/types/gas';
 
-import { gweiToWei, weiToGwei } from '@/__swaps__/utils/ethereum';
-import { addHexPrefix, convertStringToHex, toHex } from '@/__swaps__/utils/hex';
+import { gweiToWei, weiToGwei } from '@/parsers';
 import {
+  convertStringToHex,
+  lessThan,
   add,
   addBuffer,
   convertAmountAndPriceToNativeDisplay,
   convertRawAmountToBalance,
   divide,
   fraction,
   greaterThan,
-  lessThan,
   multiply,
-} from '@/__swaps__/utils/numbers';
-import { getMinimalTimeUnitStringForMs } from '@/__swaps__/utils/time';
+} from '@/helpers/utilities';
+import { addHexPrefix, toHex } from '@/handlers/web3';
 import { MeteorologyLegacyResponse, MeteorologyResponse } from '@/entities/gas';
 
 export const FLASHBOTS_MIN_TIP = 6;
 
+export const buildLocalizedTimeUnitString = ({ plural, short, unit }: { plural?: boolean; short: boolean; unit: string }) => {
+  const length = short ? 'short' : 'long';
+  const plurality = plural ? 'plural' : 'singular';
+
+  return i18n.t(`time.${unit}.${length}.${plurality}`);
+};
+
+const getHighestResolutionUnit = (timeUnitKey?: string, timeUnitValues?: { [key: string]: number }) => {
+  const highestResolutionUnit = timeUnitKey || 'seconds';
+  return {
+    unit: highestResolutionUnit,
+    value: timeUnitValues?.[highestResolutionUnit] || 0,
+  };
+};
+
+/**
+ * @desc get time string for minimal unit
+ * @param {String} [value='']
+ * @param {Boolean} [short=true]
+ * @param {Boolean} [plural=false]
+ * @return {String}
+ */
+export const getMinimalTimeUnitStringForMs = (value: number, plural?: boolean, short = true): string => {
```

Comment:
same thing here: could we just use the `getMinimalTimeUnitStringForMs` in `helpers/time` instead of moving the duplicate to this file?